game/static/game/css/board.css

```
@@ -1061,14 +1514,21 @@ body {
     gap: 12px;
 }
 
-.promo-logo span {
+.promo-logo .logo-text {
     font-family: 'Cinzel', serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Cinzel" (font-family-name-quotes)

(font-family-name-quotes)